Find a functionally equivalent code:com.android.internal.widget.LockPatternUtils.clearLock:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Clear any lock pattern or password.
 */

@@ -6,9 +6,10 @@
     saveLockPattern(null);
     setLong(PASSWORD_TYPE_KEY, DevicePolicyManager.PASSWORD_QUALITY_UNSPECIFIED);
     setLong(PASSWORD_TYPE_ALTERNATE_KEY, DevicePolicyManager.PASSWORD_QUALITY_UNSPECIFIED);
+    onAfterChangingPassword();
 }

